cinema_service/settings.py

```
@@ -86,8 +87,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB"),
+        "USER": os.environ.get("POSTGRES_USER"),
```

Same as above for `POSTGRES_USER`: if the env var is missing this will be None and DB auth will fail. Either provide a default (for dev) or validate and fail fast with a descriptive message so `wait_for_db` and migrations don't hang on ambiguous errors.